CacheMeIfYouCan/CacheMeIfYouCanTests/TestImageLoader.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  TestImageLoader.swift
+//  CacheMeIfYouCanTests
+//
+//  Created by Ellen Shapiro on 12/11/18.
+//  Copyright © 2018 Bakken & Baeck. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+class TestImageLoader {
+    
+    enum TestImage: String {
+        case burns
+        case homer
+        case smithers
+        
+        var remoteURL: URL {
+            switch self {
+            case .burns:
+                return URL(string: "https://raw.githubusercontent.com/bakkenbaeck/CacheMeIfYouCan/master/CacheMeIfYouCan/CacheMeIfYouCanTests/TestImages/burns.png")!
+            case .homer:
+                return URL(string: "https://raw.githubusercontent.com/bakkenbaeck/CacheMeIfYouCan/master/CacheMeIfYouCan/CacheMeIfYouCanTests/TestImages/homer.png")!
+            case .smithers:
+                return URL(string: "https://raw.githubusercontent.com/bakkenbaeck/CacheMeIfYouCan/master/CacheMeIfYouCan/CacheMeIfYouCanTests/TestImages/smithers.png")!
```

Comment:
Can we license these images under the same license as this library?

Reply:
They're from wikipedia, I just grabbed them and put them in our repo so we'd have a consistent URL for them and make sure they don't change on us. I'll update the license to note these are covered by the same license as they are at Wikipedia.

Reply:
As well as referencing Fair Use from Wikipedia you'll need to add reference to this license `The Simpsons ™ & © Twentieth Century Fox Film Corp. All Rights Reserved.`

I'm sure it's not an issue, but we could go for public domain images instead.

Reply:
🤔I could also just grab images from my Flickr page that I own the copyright to.